src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs

```diff
@@ -0,0 +1,108 @@
+﻿using AutoFixture;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.ProviderCommitments.Web.Controllers;
+using SFA.DAS.ProviderCommitments.Web.Models.Cohort;
+using SFA.DAS.ProviderUrlHelper;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.CohortControllerTests
+{
+    [TestFixture]
+    public class WhenGettingDetails
+    {
+        private WhenGettingDetailsTestFixture _fixture;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _fixture = new WhenGettingDetailsTestFixture();
+        }
+
+        [Test]
+        public async Task ThenViewModelShouldBeMappedFromRequest()
+        {
+            await _fixture.GetDetails();
+            _fixture.VerifyViewModelIsMappedFromRequest();
+        }
+
+        [TestCase(Party.Employer)]
+        [TestCase(Party.TransferSender)]
+        public async Task ThenViewModelIsReadOnlyIfCohortIsNotWithEmployer(Party withParty)
```

Comment:
Should this be "Is With Employer" or "Is not with provider"?

Reply:
Yes, it should be